content/topics/work/recruit-career/_index.md

```diff
@@ -0,0 +1,10 @@
+---
+ready: true
+title: Recruit Career Questionnaire
+---
+
+Complete the above mentioned Questionnaire
```

Comment:
"Complete the following..." Not "complete the above mentioned..." The questionaire is not above this statement, it follows this statement.